recipe/recipe.go

```diff
@@ -32,27 +33,21 @@ func (c *Client) GetRecipe(ctx context.Context, userAuthToken, serviceAuthToken,
 
 	b, err := ioutil.ReadAll(res.Body)
 	if err != nil {
-		return nil, &Error{
-			err:        fmt.Errorf("failed to read response body: %s", err),
-			statusCode: res.StatusCode,
-			logData: log.Data{
-				"response_body": string(b),
-			},
-		}
+		return nil, dperrors.New(
+			fmt.Errorf("failed to read response body: %s", err),
+			res.StatusCode,
+			log.Data{"response_body": string(b)})
```

Comment:
bracket should be closed on the next line (keep in line with where scope was opened)